apps/trading/client-pages/markets/top-market-list.tsx

```diff
@@ -0,0 +1,51 @@
+import { EmblemByMarket } from '@vegaprotocol/emblem';
+import type { MarketMaybeWithCandles } from '@vegaprotocol/markets';
+import { useChainId } from '@vegaprotocol/wallet-react';
+import { Link } from 'react-router-dom';
+import {
+  priceValueFormatter,
+  priceChangeRenderer,
+  priceChangeSparklineRenderer,
+} from './use-column-defs';
+import { StackedCell } from '@vegaprotocol/datagrid';
+
+export const TopMarketList = ({
+  markets,
+}: {
+  markets?: MarketMaybeWithCandles[];
+}) => {
+  const { chainId } = useChainId();
+  return (
+    <>
+      {markets?.map((market) => {
+        return (
+          <div
+            className="grid auto-rows-min grid-cols-7 gap-3 text-xs"
```

Comment:
Couple things

- The layout of the card looks too tight
- Font size looks too small

![Screenshot 2024-04-25 at 14 28 55](https://github.com/vegaprotocol/frontend-monorepo/assets/6803987/ccc3468b-ab4b-4c1d-8e5d-1595736184ba)

Reply:
ok so made few changes
use the market code and used text-sm 

<img width="1250" alt="image" src="https://github.com/vegaprotocol/frontend-monorepo/assets/16125548/ce28ce32-4e26-40b9-acd5-1743b9ceb63d">


not sure it looks better imo but the cards are really to tight because we try to fit a lot inside.
didn't change sparkline component to make it smaller or with adjustable width... 

i even added a breakpoint to make those at the beginning smaller after a certain breakpoint 
<img width="1250" alt="image" src="https://github.com/vegaprotocol/frontend-monorepo/assets/16125548/2680780a-ffb1-4c6a-8a7d-f33e4824f0f7">